contracts/ClaimsManager.sol

```diff
@@ -92,6 +97,13 @@ contract ClaimsManager is
         _setClaimantResponsePeriod(_claimantResponsePeriod);
     }
 
+    function setApi3ToUsdReader(address _api3ToUsdReader) external override {
+        require(manager == msg.sender, "Sender not manager");
```

Comment:
I figured this is a bit similar to `setApi3Pool()` in that it's supposed to be set at deployment and not changed again. This implies that there is a semantic difference between the manager and the admin. In my mind admin is manages owners of roles, so just as it can grant or revoke roles, it can also manage the parameters related to these roles. I guess I will think a bit more about this. "Admin==manager in the context of authority over a particular contract" requires less mental gymnastics.

Reply:
I ended up allowing admin to do whatever the manager can do